src/table_builder.cc

```diff
@@ -115,16 +106,27 @@ void TitanTableBuilder::Add(const Slice& key, const Slice& value) {
                         index.file_number, get_status.ToString().c_str());
       }
     }
-    if (builder_unbuffered()) {
-      base_builder_->Add(key, value);
-    } else {
-      std::unique_ptr<BlobFileBuilder::BlobRecordContext> ctx =
-          NewCachedRecordContext(ikey, value);
-      blob_builder_->AddSmall(std::move(ctx));
-    }
+    AddSmall(key, ikey, value);
   } else {
-    assert(builder_unbuffered());
+    // Mainly processing kTypeMerge and kTypeBlobIndex in both flushing and
+    // compaction.
+    AddSmall(key, ikey, value);
+  }
+}
+
+void TitanTableBuilder::AddSmall(const Slice& key,
```

Comment:
The naming is still confusing. How about renaming the old `AddToBaseTable` to `AddBlobResultsToBase`, and `AddSmall` to `AddBase` (symmetric to `AddBlob`).

Reply:
ok, I will do these changes.